src/common/utility.js

```diff
@@ -89,8 +89,9 @@ const deriv_app_url = 'https://app.deriv.com'
 const deriv_bot_app_url = 'https://app.deriv.com/bot'
 const smarttrader_url = 'https://smarttrader.deriv.com'
 const binary_url = 'https://binary.com'
-const affiliate_signup_url = 'https://login.deriv.com/signup.php'
-const affiliate_signin_url = 'https://login.deriv.com/signin.php'
+const deriv_cookie_domain = 'deriv.com'
+const affiliate_signup_url = 'https://login.binary.com/signup.php'
```

Comment:
?
shouldn't it be login.deriv.com?, why do we change back to binary?

Reply:
Ah I think the branch is outdated. Will push a fix.